core/src/main/java/com/yahoo/oak/InternalOakHash.java

```diff
@@ -705,30 +666,10 @@ protected void initState() {
             return hashArray.getNextChunk(current, lastKeyHash, hashValid);
         }
 
-        @Override
-        protected void advanceState() {
```

Comment:
Was this method just moved to `InternalOakBasics`? Or there are changes we should address?

Reply:
yes, moved to the parent class.